lineage/states/StateDistributionGamma.py

```diff
@@ -48,16 +48,22 @@ def pdf(self, x):  # user has to define how to calculate the likelihood
         # In our example, we assume the observation's are uncorrelated across the dimensions (across the different
         # distribution observations), so the likelihood of observing the multivariate observation is just the product of
         # the individual observation likelihoods.
-        ll = np.zeros(x.shape[0])
+        ll_b = np.zeros(x.shape[0])
+        ll_g = np.zeros(x.shape[0])
 
         # Update for observed Bernoulli
-        ll[np.isfinite(x[:, 0])] += sp.bernoulli.logpmf(x[np.isfinite(x[:, 0]), 0], self.params[0])
+        ll_b[np.isfinite(x[:, 0])] += sp.bernoulli.logpmf(x[np.isfinite(x[:, 0]), 0], self.params[0])
 
         # Update uncensored Gamma
-        ll[x[:, 2] == 1] += sp.gamma.logpdf(x[x[:, 2] == 1, 1], a=self.params[1], scale=self.params[2])
+        ll_g[x[:, 2] == 1] += sp.gamma.logpdf(x[x[:, 2] == 1, 1], a=self.params[1], scale=self.params[2])
 
         # Update censored Gamma
-        ll[x[:, 2] == 0] += sp.gamma.logsf(x[x[:, 2] == 0, 1], a=self.params[1], scale=self.params[2])
+        ll_g[x[:, 2] == 0] += sp.gamma.logsf(x[x[:, 2] == 0, 1], a=self.params[1], scale=self.params[2])
+
+        # remove dead cells
+        ll_g[x[:, 0] == 0] = 0
+
+        ll= ll_b + ll_g
 
         return np.exp(ll)
```

Comment:
Is this a correct change for the pdf function? @aarmey

Reply:
For the dead cells we only have the bernoulli likelihood, so I set the likelihood calculated from the gamma to zero.